backend/auth/jwt.py

```
@@ -1,43 +1,45 @@
-            }
+                "verify_aud": False,
+            },
         )
-
         return payload
 
     except ExpiredSignatureError:
         raise TokenExpiredError()
     except JWTError as e:
         raise InvalidTokenError(detail=f"Invalid token: {str(e)}")
     except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)